v3/docs/gs/params.md

```diff
@@ -7,9 +7,8 @@ or *runtime* parameters, which apply each time an application
 initializes the HSE library or opens a KVDB or KVS.
 
 In addition to the detailed descriptions below, you can find parameter
-usage examples in the [API reference](../api/c/Pages/examples.md),
-[CLI reference](cli.md), and HSE
-[sample programs](https://github.com/hse-project/hse/tree/master/samples).
+usage in the [CLI reference](cli.md) and HSE
+[sample programs](https://github.com/hse-project/hse/tree/v3.0/samples).
```

Comment:
Why is it ".../tree/v3.0/samples" here and ".../tree/r2/samples" in v2/docs/params.md?

Reply:
Because hse 3.0 hasn't been tagged yet, but I guess it is just a link so it doesn't matter

Reply:
Right, let's make them consistent.  Before this update it just pointed to latest for all doc versions, but actually that wasn't correct given changes between major versions.